Add 2023-11-01 agenda and minutes (#591)
solid/specification: meetings/2023-11-01.md

# W3C Solid Community Group: Weekly

* Date: 2023-11-01T14:00:00Z
* Call: https://meet.jit.si/solid-cg
* Chat: https://matrix.to/#/#solid_specification:gitter.im
* Repository: https://github.com/solid/specification

> * Status: Draft

Author: ```suggestion
* Status: Published
```

* ...: orgs will be ordered by number of participants; those with more than one will be mentioned on the mailing list so that a representative can be named
* eP: another option may be to work with W3C to enforce certain authorization rules within their system. There are ways to accomplish this, but it will likely be a lot of manual work. Personally, I don't see a problem with this type of enforcement
* ...: I don't want to bite off to much with this process
* SC: we're using as much as we can, given what W3C offers
* ...: The questionnaire is limited to CG participants. The W3C is curious to see how this will work with a CG
* ...: Other CGs have a similar process
* ...: The W3C will announce the winners on the mailing list
* ...: We use W3C infrastructure to take advantage of existing W3C accounts
* AC: I was wondering if you have a timeframe in mind for when we will initiate voting.
* SC: once we have eligible voters ready. Let's say within 1 week to announce to mailing list
* AC: I find it helpful if we have dates, we've been talking about the vote for a while now.
* SC: I think this is the same issue as we had with spec releases.

> * AC: From my own experience I'm really confused when it comes to the timeframe. A lot of folks would love to see it happen. Many folks already nominated themselves. If we have some dates it would be a good motivation to make it happen.

Reviewer: ```suggestion
* AC: From my own experience, I'm really confused when it comes to the timeframe. A lot of folks would love to see it happen. Many folks already nominated themselves. If we have some dates, it would be a good motivation to make it happen.
```

> * SC: Agree with you but dates haven't been particularly helpful in a number of cases, as in there is apathy or accountability in the CG.

Reviewer: ```suggestion
* SC: Agree with you, but dates haven't been particularly helpful in a number of cases, as in there is apathy or accountability(?) in the CG.
```

* TBL: CG process historically was ..., people don't come to the meetings because..., 

> * SC: An example about dates: when I made [PR to release SP v0.10](https://github.com/solid/specification/pull/492) only Kjetilk reviewed it on time. It wasn't just about "CG process". You say things are bureaucratic sometimes but tell me what prevented people, including the other co-editors to review that in the first place? And ditto Solid-OIDC issues that I've created that the editors did not take on entirely - issues still sitting around. I can't fire editors in the CG or something. Then what right? How about the CG charter? People that have opinions on it from the sidelines literally did not contribute. One shouldn't complain about process if they're not contributing when they had all the time in the world and chance to do so in the first place. The TR page used to have "expected dates". I went into meetings and chats and everywhere to get people to commit to dates (whatever they wanted). It didn't happen. So, coming up with dates is not the issue.

Reviewer: ```suggestion
* SC: An example about dates: when I made [PR to release SP v0.10](https://github.com/solid/specification/pull/492), only Kjetilk reviewed it on time. It wasn't just about "CG process". You say things are bureaucratic sometimes but tell me what prevented people, including the other co-editors, to review that in the first place? And ditto Solid-OIDC issues that I've created that the editors did not take on entirely — issues still sitting around. I can't fire editors in the CG or something. Then what, right? How about the CG charter? People that have opinions on it from the sidelines literally did not contribute. One shouldn't complain about process if they're not contributing when they had all the time in the world and chance to do so in the first place. The TR page used to have "expected dates". I went into meetings and chats and everywhere to get people to commit to dates (whatever they wanted). It didn't happen. So, coming up with dates is not the issue.
```

* eP: someone could invite 10-15 people to the CG as individual contributors today. it is easy to game the system. We shouldn't try to over-engineer the system
* SC: Right. It is also taking advantage of W3C infra and support as much as possible.
* AC: I feel like the conversation is going in a lot of different directions. Dates about previous releases of spec and editors doesn't seem to be in place. I'm just asking about expectations for the vote, talking about all the history doesn't help. We should just have a timeframe.
* ...: By the 15th of November we will have the list of candidates and start voting?

> * SC: No we can start nomination period after 15th, two weeks after that we could vote so Nov 29th.

Reviewer: ```suggestion
* SC: No, we can start nomination period after 15th; two weeks after that, we could vote, so Nov 29th.
```

* AC: I think we already had good list of candidates. We could have process of nomination in parallel.

> * SC: We wrote down the process and asked for feedback on the process. It feels like asking to change it in the last minute. I want to make an announcement to invite people to nominate themselves.

Reviewer: ```suggestion
* SC: We wrote down the process and asked for feedback on it. This feels like asking to change it in the last minute. I want to make an announcement to invite people to nominate themselves.
```

* MdJ: Aren't the participants of the CG eligible candidates?
* SC: Not everyone, if a company is affiliated with more people, ...

> * TBL: I see it example of being overly bureaucratic, the process may not be perfect ...

Reviewer: ```suggestion
* TBL: I see it as an example of being overly bureaucratic; the process may not be perfect ...
```

* SC: Maybe we can shorten it one week, I'm hoping that we can have the eligible list sorted out soon.
* TBL: Why do you need to have it sorted out before you send the call for nominations?
* SC: The idea is that those who are nominating are eligible.
* MdJ: If you get 2 candidates from the same company they could just need to choose one
* TBL: everyone can nominate...

> * AC: I know that Pavlik had tentative nomination, I would like to nominate him. Whatever process is relevant I would like to do that. Processes are great, they should clarify how things supposed to happen. When I look at the process for voting I'm just confused. For me that seems like something is not working there.

Reviewer: ```suggestion
* AC: I know that Pavlik had tentative nomination; I would like to nominate him. Whatever process is relevant I would like to do that. Processes are great; they should clarify how things supposed to happen. When I look at the process for voting, I'm just confused. For me, that seems like something is not working there.
```

> * ...: In my experience, when there is confusion about something, I would see a leader coming to help clarifying it.

Reviewer: ```suggestion
* ...: In my experience, when there is confusion about something, I would see a leader come to help clarify it.
```

* SC: Perhaps people that haven't contributed to processes shouldn't complain.
* SC: We haven't come to determine the list of eligible participants. The process I came up with was coordinated with W3C staff. I'm happy to improve the process.

> * AC: I think there are 2 levels of clarity, one is something written down. But also ensuring that it happens in understandable way is also important.

Reviewer: ```suggestion
* AC: I think there are 2 levels of clarity; one is something written down. But ensuring that it happens in understandable way is also important.
```

* eP: it sounds like we can have this running in parallel. But that process is locked so it may be hard to modify
* MdJ: I'm ok with waiting another 2 weeks if needed

> * AC: It's fine with me, that's the first time I hear a concrete dates.

Reviewer: ```suggestion
* AC: It's fine with me, that's the first time I hear concrete dates.
```
